improver/wind_calculations/wind_components.py

```diff
@@ -139,30 +141,28 @@ def calc_true_north_offset(reference_cube):
         return angle_adjustment
 
     @staticmethod
-    def resolve_wind_components(speed, angle, adj):
+    def resolve_wind_components(
+        speed: Cube, angle: Cube, adj: ndarray
+    ) -> Tuple[Cube, Cube]:
         """
         Perform trigonometric reprojection onto x and y axes
 
         Args:
-            speed (iris.cube.Cube):
+            speed:
                 Cube containing wind speed data
-            angle (iris.cube.Cube):
+            angle:
                 Cube containing wind directions as angles from true North
-            adj (numpy.ndarray):
+            adj:
                 2D array of wind direction angle adjustments in radians, to
                 convert zero reference from true North to grid North.
                 Broadcast automatically if speed and angle cubes have extra
                 dimensions.
 
         Returns:
-            (tuple): tuple containing:
-                **u_speed** (iris.cube.Cube):
-                    Cube containing wind vector component in the positive
-                    x-direction
-
-                **v_speed** (iris.cube.Cube):
-                    Cube containing wind vector component in the positive
-                    y-direction
+            - Cube containing wind vector component in the positive
+              x-direction
+            - Cube containing wind vector component in the positive
+              y-direction
```

Comment:
```suggestion
            - Cube containing wind vector component in the positive
              x-direction u_speed
            - Cube containing wind vector component in the positive
              y-direction v_speed
```
Now I'm just being consistent with my dim recollections of comments from hours ago.... 89/96 files. So close 😄